<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showAllInfoDialog</class>
 <widget class="QDialog" name="showAllInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="stuInfoWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>641</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>专业</string>
    </property>
   </column>
  </widget>
  <widget class="QTabWidget" name="searchTabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>641</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="searchByNameTab">
    <attribute name="title">
     <string>通过名称查找</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 宋体 Std L</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入学生姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找(&amp;A)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchByIdTab">
    <attribute name="title">
     <string>通过学号查找</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 宋体 Std L</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入学生学号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idLineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchByIdButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找(&amp;A)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchByDepartmentTab">
    <attribute name="title">
     <string>通过专业查找</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 宋体 Std L</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入专业名称：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="departmentNameLineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找(&amp;A)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示所有学生(&amp;S)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchStuDetailButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看选中学生详细信息(&amp;O)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delStuButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中学生信息(&amp;I)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加学生(&amp;D)</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通过上方选项卡搜索学生信息。双击学生信息查看详细信息，并可以为其添加兴趣修改信息。或者选择移除学生信息。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
